frameworks/Python/tornado/benchmark_config.json

```diff
@@ -19,14 +21,40 @@
       "webserver": "Tornado",
       "os": "Linux",
       "database_os": "Linux",
-      "display_name": "Tornado-raw",
+      "display_name": "Tornado-Py2-Mongo",
```

Comment:
Please change this to "Tornado" (if it is not a raw database test any longer).

The `py2` and `mongo` metadata is captured and displayed elsewhere.